libs/ssr/express/src/response.token.ts

```diff
@@ -0,0 +1,14 @@
+import { Response } from 'express';
+
+import { createSingleInjectionToken } from '@daffodil/core';
+
+export const {
+  /**
+   * A token to hold the express response during SSR.
+   */
+  token: DAFF_SSR_EXPRESS_RESPONSE,
+  /**
+   * Provider for `DAFF_SSR_EXPRESS_RESPONSE`.
+   */
+  provider: provideDaffSsrExpressResponse,
+} = createSingleInjectionToken<Response>('DAFF_SSR_EXPRESS_RESPONSE');
```

Comment:
I think this response type should be our own and not explicitly from express.

We should support solely a tiny subset of the `Response` methods that are relevant for Angular developers. I'd suggest starting with only the response methods that we currently use and omitting the rest.

At the outset, I think:

```ts
export interface Response {
  get(header: string): string;
  set(headers: Dict<string, Array<string> | string>): void;
  append(string, Array<string> | string>): void;
  status(code: number): Response
}
```

This would replace `DaffSsrHeaderExpressService`

Reply:
What's your preference for the translation layer between our type and the express response?

Reply:
I think this is simple enough:

```
import { Response as ExpressResponse } from 'express';

export interface DaffResponse {
  get(header: string): string;
  set(headers: Record<string, Array<string> | string>): void;
  append(header: string, value: Array<string> | string): void;
  status(code: number): Response
}

export class Response implements DaffResponse {
  constructor(
     private response: ExpressResponse,
  ) {}

  get(header: string): string {
    return this.response.get(header);
  } 

  set(headers: Record<string, string | string[]>): void {
    this.response.set(headers);
  }

  append(header: string, value: string | string[]): void {
    this.response.append(header, value);
  }

  status(code: number): Response {
    this.response.status(code);
    return this;
  }
}
```

Reply:
Does `status` need to be chainable?

Reply:
changed